src/routes/error-router.ts

```diff
@@ -63,13 +69,27 @@ ErrorRouter.use((err: Error, req: Request, res: Response, next: NextFunction) =>
 		"Not Found": 404
 	};
 
-	const messages = {
-		[Errors.MISSING_JIRA_HOST]: "Session information missing - please enable all cookies in your browser settings.",
-		[Errors.IP_ALLOWLIST_MISCONFIGURED]: `The GitHub org you are trying to connect is currently blocking our requests. To configure the GitHub IP Allow List correctly, <a href="${envVars.GITHUB_REPO_URL}/blob/main/docs/ip-allowlist.md">please follow these instructions</a>.`
-	};
+	let message: string | null = null;
+	let ctaUrl: Cta | null = null;
+	switch (err.message) {
+		case Errors.MISSING_JIRA_HOST:
+			message = "Session information missing - please enable all cookies in your browser settings.";
+			break;
+		case Errors.MISSING_GITHUB_APP_NAME:
+			message = "There was a problem creating your GitHub App. Please make sure you filled the GitHub App name and try again.";
+			ctaUrl = {
+				text: "Retry",
```

Comment:
Again. It's bad practice to define the text on the backend. We shouldn't be setting this here.

Reply:
Retry text removed from backend and hard coded in frontene